.azure-pipelines/azure-pipelines-win.yml

```diff
@@ -62,6 +62,10 @@ jobs:
     - script: conda.exe config --add channels conda-forge
       displayName: configure conda channels
 
+    # Add the cahnnel for gmp & mpfr
+    - script: conda.exe config --add channels salilab
```

Comment:
Please also add these packages to conda-forge.

Reply:
will do. in the meantime i have removed the windows dependencies i was trying to get from that channel (and the corresponding functionality of the package)